packages/ui/app/src/api-page/types/object/ObjectProperty.tsx

```diff
@@ -16,10 +17,11 @@ import { TypeShorthand } from "../type-shorthand/TypeShorthand";
 export declare namespace ObjectProperty {
     export interface Props {
         property: FernRegistryApiRead.ObjectProperty;
+        anchor?: string;
```

Comment:
why optional?

Reply:
Since the anchor element is absolutely positioned and falls a little outside of the component itself I thought we may want to support a version that doesn't have the anchor. But it looks like the component is currently used in one place only so I probably can make it required. Shall I?

Reply:
Yeah I would also default to required to make sure we're not missing anywhere we're supposed to be passing in. Any usually I'll do `anchor: string | undefined` for the same reason